anyway/flask_app.py

```diff
@@ -1178,7 +1178,7 @@ def load_identity_when_session_expires():
 )
 app.add_url_rule("/api/news-flash", endpoint=None, view_func=news_flash, methods=["GET"])
 
-app.add_url_rule("/api/news-flash-v2", endpoint=None, view_func=news_flash_v2, methods=["GET"])
+app.add_url_rule("/api/v1/news-flash", endpoint=None, view_func=news_flash_v1, methods=["GET"])
 
```

Comment:
Same here: 
Regarding API versioning and function names.
I think it is not a good idea to change function names. If UI wants to move API version and stay with same API URL, then I think we should only change the end point definition to point to a new function.